CONTRIBUTING.md

```diff
@@ -110,9 +110,13 @@ Some platforms require additional setup. See below for details.
 `libsecret` and `glib` are required for some plugins. They can be installed on Debian based linux distributions such as Ubuntu by running the command below.
 
 ```sh
-sudo apt-get update && sudo apt-get install libsecret-1-dev libglib2.0-dev
+sudo apt update && sudo apt install libsecret-1-dev libglib2.0-dev
```

Comment:
Did this command change in the latest versions of Ubuntu?

Reply:
No, they're different frontends for the same package manager - `apt` is facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))_

Reply:
No, they're different frontends for the same package manager - apt is de facto (note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))

Reply:
No, they're different frontends for the same package manager - `apt` is de facto (note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))

Reply:
aptNo, they're different frontends for the same package manager - `apt` is de facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience)_)

Reply:
No, they're different frontends for the same package manager - `apt` is de facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))_

Reply:
thanks for the link, lgtm